src/containers/chat/index.js

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
-import { Route } from 'react-router-dom'
-import Login from '../login'
-import Register from '../register'
 import Body from '../body'
+import Account_Screen from './account_screen'
+import Existing_Chat from './Existing_chat'
+import Account_Screen_New from './account_screen_new'
+import New_Chat from './New_Chat'
+import TextField from 'material-ui/TextField';
+import { withStyles } from 'material-ui/styles';
+import Paper from 'material-ui/Paper';
+import Grid from 'material-ui/Grid';
+import './index.css'
+import Button from 'material-ui/Button/Button';
 
 const Chat = () => (
     <div>
-        <p>Chat</p>
+        <div className="msgContainer">
+            <Account_Screen />   
+        </div>
+        <div>

```

Comment:
what is that div for? surely it's just polluting the code base?

Reply:
Please resolve this before the PR can be approved.